modules/zstd/metadata.json

```diff
@@ -1,17 +1,18 @@
 {
-  "homepage": "https://github.com/facebook/zstd",
-  "maintainers": [
-    {
-      "email": "bcr-maintainers@bazel.build",
-      "name": "No Maintainer Specified"
-    }
-  ],
-  "repository": [
-    "github:facebook/zstd"
-  ],
-  "versions": [
-    "1.5.5",
-    "1.5.5.bcr.1"
-  ],
-  "yanked_versions": {}
+    "homepage": "https://github.com/facebook/zstd",
+    "maintainers": [
+        {
+            "email": "bcr-maintainers@bazel.build",
```

Comment:
Could one of you (or more) add themselves as maintainers? You don't have to be a maintainer of the upstream repo for that.

Reply:
Added myself.